bigbluebutton-html5/imports/ui/components/audio/audio-modal/container.jsx

```diff
@@ -97,7 +98,7 @@ export default lockContextContainer(withModalMounter(withTracker(({ mountModal,
     combinedDialInNum,
     audioLocked: userLocks.userMic,
     joinFullAudioImmediately: !listenOnlyMode && skipCheck,
-    joinFullAudioEchoTest: !listenOnlyMode && !skipCheck,
+    joinFullAudioEchoTest: (!listenOnlyMode && !skipCheck) || (Storage.getItem('audioDidInit')),
```

Comment:
`joinFullAudioEchoTest` is being passed as `null` to `AudioModal` on first join

![Screenshot from 2021-02-01 17-05-06](https://user-images.githubusercontent.com/3728706/106513169-58fe9380-64b1-11eb-899c-3ab806b09978.png)

Reply:
fixed in https://github.com/bigbluebutton/bigbluebutton/pull/11284/commits/85cc75718ad6528bf068fa4b78eaf91f566f651d